orion/sites/admin/src/home/home.js

```diff
@@ -2,30 +2,210 @@
+            });
+            this.state.switch = false;
+        }
+    }
+
+    onClickSessions() {
+        this.setState({ noUnpub: "" });
+        let sessions = this.state.sessions.map((row, index) => {
+            return <DashboardRow key={index} row={row} />;
+        });
+        this.setState({ row: sessions });
+        this.setState({ selection: "sessions" });
+        this.setState({ num: sessions.length });
+        this.state.switch = true;
+        if (sessions.length == 0) {
+            this.setState({
+                noUnpub: "All items are published! Your website is up to date!",
+            });
+            this.state.switch = false;
+        }
+    }
+
     render() {
+        let unpubMessage = <div> </div>;
+        if (this.state.switch) {
+            unpubMessage = (
+                <h4>
+                    {" "}
+                    You have {this.state.num} unpublished {this.state.selection}{" "}
+                </h4>
+            );
+        }
         return (
             <div id="view-home">
                 <div id="view-content">
                     <h2 id="unpublished-heading"> Unpublished Content </h2>
                     <ul>
-                        <li>Programs</li>
-                        <li>Locations</li>
-                        <li>Achievements</li>
+                        <button onClick={this.onClickPrograms}>Programs</button>
+                        <button onClick={this.onClickClasses}>Classes</button>
+                        <button onClick={this.onClickLocations}>
+                            Locations
+                        </button>
+                        <button onClick={this.onClickAchievements}>
+                            Achievements
+                        </button>
+                        <button onClick={this.onClickSemesters}>
+                            Semesters
+                        </button>
+                        <button onClick={this.onClickSessions}>Sessions</button>
                     </ul>
                     <h2> Registrations </h2>
                     <ul>
-                        <li>New Users</li>
-                        <li>Questions</li>
-                        <li>Complaints</li>
+                        <button>New Users</button>
+                        <button>Questions</button>
+                        <button>Complaints</button>
                     </ul>
                 </div>
                 <div id="box-and-button">
-                    <div className="boxed">text</div>
+                    <div className="boxed">
+                        {unpubMessage}
+                        {this.state.row}
+                        {this.state.noUnpub}
+                    </div>
                     <button id="go-to-page">Go to Page</button>
                 </div>
             </div>
         );
     }
 }
+
+class DashboardRow extends React.Component {
+    render() {
+        const row = this.props.row;
```

Comment:
Consider another way of doing this. Instead of passing an entier `row` to `DashboardRow`, pass in just a single `title` string variable and display it below. In this scenario, you only have to worry about displaying one variable.

Above, in your `DashboardRow` creating map part, you can specify which variable of the object you want to pass.
```
let semesters = this.state.semesters.map((row, index) => {
            return <DashboardRow key={index} title={row.semesterId} />;
});
```

I think the concern with the current approach is you end up risking displaying multiple things when you don't intend to.